.github/workflows/cron.yml

```
@@ -0,0 +1,27 @@
+name: Cron
+
+on:
+  schedule:
+    - cron:  '0 6 * * *'  # Daily 6AM UTC build
+
+env:
+  REF: master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  # No compile-deps on cron job.
+  # TODO: Apply run to all active branches by keeping workflows like cron-38.yml etc.
+
+  lint:
+    name: Lint
+    uses: aio-libs/aiohttp/.github/workflows/sub-lint.yml@master
+    with:
+      ref: ${{ env.ref }}
```

Would something like
```suggestion
      ref: ${{ github.event.ref }}
```
work?

---

Or maybe
```suggestion
      ref: ${{ github.event.repository.default_branch }}
```

---

Thanks!
I think the `master` constant should work here.
Not existing yet `cron-39.yml` file will contain `'3.9'` instead